java/change_atmosphere_effect/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,9 @@
+<resources>
+    <string name="app_name">change_atmosphere_effect</string>
+    <string name="elevation_image_service">
+        https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer</string>
+    <string name="noAtmosphereEffect">No Atmosphere Effect</string>
```

Comment:
use "this_kind_of_naming" for strings (I don't know why, it just seems to be the Android thing to do)